bundles/org.openhab.binding.miio/src/main/java/org/openhab/binding/miio/internal/MiIoDevices.java

```diff
@@ -90,6 +90,18 @@
     GATEWAY3("lumi.gateway.v3", "Mi Smart Home Gateway v3", THING_TYPE_BASIC),
     GATEWAY_MGL3("lumi.gateway.mgl03", "Xiaomi Mi Mijia Gateway V3 ZNDMWG03LM", THING_TYPE_BASIC),
     HUMIDIFIER("zhimi.humidifier.v1", "Mi Humdifier", THING_TYPE_BASIC),
+    HUAYI_LIGHT_ARI013("huayi.light.ari013", "huayi.light.ari013", THING_TYPE_BASIC),
+    HUAYI_LIGHT_ARIES("huayi.light.aries", "huayi.light.aries", THING_TYPE_BASIC),
+    HUAYI_LIGHT_PEG091("huayi.light.peg091", "huayi.light.peg091", THING_TYPE_BASIC),
+    HUAYI_LIGHT_PEG093("huayi.light.peg093", "huayi.light.peg093", THING_TYPE_BASIC),
+    HUAYI_LIGHT_PIS123("huayi.light.pis123", "huayi.light.pis123", THING_TYPE_BASIC),
+    HUAYI_LIGHT_PISCES("huayi.light.pisces", "huayi.light.pisces", THING_TYPE_BASIC),
+    HUAYI_LIGHT_TAU023("huayi.light.tau023", "huayi.light.tau023", THING_TYPE_BASIC),
+    HUAYI_LIGHT_TAURUS("huayi.light.taurus", "huayi.light.taurus", THING_TYPE_BASIC),
+    HUAYI_LIGHT_VIR063("huayi.light.vir063", "huayi.light.vir063", THING_TYPE_BASIC),
+    HUAYI_LIGHT_VIRGO("huayi.light.virgo", "huayi.light.virgo", THING_TYPE_BASIC),
+    HUAYI_LIGHT_WY("huayi.light.wy", "huayi.light.wy", THING_TYPE_BASIC),
+    HUAYI_LIGHT_ZW131("huayi.light.zw131", "huayi.light.zw131", THING_TYPE_BASIC),
```

Comment:
Don't these need proper labels?

Reply:
You are fully right.
I made a bigger rename now, as I finally figured out how to pull the proper names of the devices from the cloud